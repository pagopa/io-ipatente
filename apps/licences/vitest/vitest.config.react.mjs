/// <reference types="vitest" />
/// <reference types="vite/client" />

import react from "@vitejs/plugin-react";
import { loadEnv } from "vite";
import tsconfigPaths from "vite-tsconfig-paths";
import { configDefaults, defineConfig } from "vitest/config";

export default defineConfig({
  plugins: [react(), tsconfigPaths()],
  test: {
    coverage: {
      exclude: [
        ...configDefaults.coverage.exclude,
        "**/__tests__",
        "src/generated/**",

        // BE
        "src/app/*",
        "src/config/*",
        "src/lib/bff/*",
      ],
      ignoreEmptyLines: true,
      include: ["src/**/*.{tsx,ts}"],
      reporter: ["text", "html"],
    },
    css: true,
    env: loadEnv("test", process.cwd(), ""),
    environment: "jsdom",
    exclude: [
      ...configDefaults.exclude,
      "./vitest/vitest.setup.react.ts",
      "./vitest/vitest.setup.node.ts",
    ],
    globals: true,
    include: ["./src/**/*.test.tsx"],
    setupFiles: "./vitest/vitest.setup.react.ts",
    typecheck: {
      ignoreSourceErrors: true,
    },
  },
});
