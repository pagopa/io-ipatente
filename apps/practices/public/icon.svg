<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.dev/svgjs" width="32" height="32" viewBox="0 0 32 32"><image width="32" height="32" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAemgAAHpoB/G8o+AAAABx0RVh0U29mdHdhcmUAQWRvYmUgRmlyZXdvcmtzIENTNui8sowAAAcsSURBVFiFpZd7cFXVFcZ/+5xzn7kJSS4hIXBJDJABgSgPA+10KAW0BTrVmNTWAoKEvpzCTB/WKRUf07FOlSpqaWeqtCVIcUqUFgudBtQgFEZKeBgUQklJREISEpKbm/s85+zdP24aHnmC35/n7P2tb397rb3XFkophou05ZXFwELgbmAqkAkooAU4BuwFqrq2lp0ZLqcYSkDa8koDKAWWAcXAqCE4PwEOCMEfgxVl73wmASMeqlykFL8E7hyKaABUCcG6YEVZzU0JyFi5U0gpX1JKrbnFwNfCBh7t2lr24rAEpC2vdAOVwJJbiaYUJCyJVAqHrmHoAgBNiFc6K0rX3jheu05N3q+5leBCgGlLQuEEkYRFXpaXKYE0XA6NUNQCAVKpNRkr3uzjwnUOZKx48yVbqj4qB4NUinDYJCvDzdK5t3H/7LGkpzgIZHjojpg8tu0k2w9fwOc2EICuiW9f2VL6Wh8B/offWmhZcu9wi1IIiCZsLFtRvqCAZx4swqFrBCMmgZFe9p5rp6o1RI7DYPPrJ/mkLYLD0BCCiMvQi1r/UFIPYABMWrNHl1JtlIAYZvBQ1CLT5+TV791FyewA/20Nk+KEwEgvz/7lFE/WNiPHpYImGCOu7rVSeC0pnyNZ2snvndH4fVKpKcMN3hUxGev38O5T8ymZHaBifwMfXwiSne7mZ9tOsn7LcWSqAzwOSNh0dMbQjaQEBUhJSXb5zim9AixblctheC8EdEctRmd4+Of6eRTlpVPf0k3h6FS+OjOXP1Wf5/kdp/Dm+hCZHlAKPWph2FfJBSCVEsDDAPrfuufMCkbMdbZS7qEcSFgSj6Gx++fzuDM/g63vN7C7polV8wuovxSi9LkDKKlQ+elY2SmAwNEcxmiLgn49u0Ck1dS3b9VagrHFlpTpQwWXCuIJm03fLWb2RD9x0+bouXbWld4OwNotx+kMxXH6nJi5vuQEKdG74n0SSykQgomnL4bmGn6f8xvdMWtI68OhBN9fUsjyuXlE4zYb3j7D8w9Nx2loVP6njT2aQpuXR1gXSLcDJAgzKUBpog9fwpKu1mBshtEVtSYOGhzojllMzk/nV8vuAOCR144SNyUOXcOUiieON0G6G9mTaEgFhobeHkeLWShd65d7jN/zZSMYMR2DCbCUQih4edUMUj0OumMW0YTNK+UzEQJ2HbnI6V11pLgNrBwf8fHpvcr19mhSjN4/ty60QP/SrkEklOA790xgYVEOAFvfb+DJB6aSleZCAb/ZcxYtbKKHTawM91XbTInRER1w9QCGLgzNn+oasABjCZtAjo9ffHMaAHuONfHI7/+Ns8fqQ3VtHDjdSopLx8xOwfZ7wFaga+hdcbSwmdzwAeB1GV3auCxvR38KlAIzbvPs0jvwp7oAWP9GLeULChif7QNg8756bFMidI1EIO3qZAFGexRhywGPVqXAaWgfaJk+Z9WNYwTQHTFZXDyGpXPzAfjH8SaOn7/CT+9Nlt2ljih/r2nC49Ax/R7sTA/YMklgqeT+a/3bb0tFmteBP9VZrU0bl/5Wps8prWtOK9OWpHodbFgxvffb49trKZkdoDA3FYAdhy9w+UrygkkE0kATSbtdOnoojh5K9Cm/q6tXpLqN5rWLCo9oTz0wde+oEe462XMrCiAaMXmsZDKTxyRt3fdhM8fq2lh3/+29JH8+2IiuCcxRXuzsFDAlImajt0VxNgQRSg1q/23ZKUeLJ/pPG0DnnMKRfz3bFJpsK0U4ZlE0wc9Pvjapd8LTOz7ivi+MY2ZBJgBVJ5upqb+Cx+3ABtynLqMH44iYhbAkQjFg9ltS4k91sbAo53Xo6QcutEfyyjYcrKupv+Jy6BrvPb2AOYV+4qbk3dpmFj9TzfJ5BYxKc3GisZPaxk5CUQtDFwhbgVRJu7UeDwc516OmzT1FOQ27131xEhA3AAJ+b2NJ8djfnr0Y+uHIES72fdjMC2+fob6lm3OXQqR4HLzxr0bMmIVw6LgdOg5doACliz4XzUCIW5L8rBRWzit4HIj3OtAD3+rfHTmxpfr8eMuyQQkMh4bL0NAGSKabgS0Vhq7xg0UT9zzzYFFvz3ldT5iw5PRvbTx0cOcHn3q9LmOwM+Smg9u2onxhQeOm1bOKgdb//7suU5yGdvyFldPLSucE4tGEjTWcLmUIJCyJAlbfPf7SptWzvnJtcBjgYRIz7fmPVpzYvu1Aw6iOsInvFtyQShGOWozLSqF8QcGpJ74+tRQ4e+O4wZ5mBVuqz79csb9hyaG6y8QSNh6njjHI5QLJFcdNmxFeJwumZbNqfsHmJTNzfwwE+xs/5OM0Ztr3vrq3fu07tS2fP9HQ4b7cFUcToo8jUikEkJvpZWZBZnDxjNFVy+bmvwgcHox/SAHXoOy9Uy0/OnKu/XP7P7rMx58G6Qgn0BBkjXBRlJfOl6Zlq7vGZ+4qnuDfCFQPh/R/2nEhVl+PGw8AAAAASUVORK5CYII="></image><style>@media (prefers-color-scheme: light) { :root { filter: none; } }
@media (prefers-color-scheme: dark) { :root { filter: none; } }
</style></svg>