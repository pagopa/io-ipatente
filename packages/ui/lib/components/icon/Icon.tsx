import { SvgIconProps } from "@mui/material/SvgIcon";

import IconAirplane from "./svg/IconAirplane";
import IconAlarm from "./svg/IconAlarm";
import IconAnchor from "./svg/IconAnchor";
import IconBus from "./svg/IconBus";
import IconBusStation from "./svg/IconBusStation";
import IconCar1 from "./svg/IconCar1";
import IconCar2 from "./svg/IconCar2";
import IconCarCollision from "./svg/IconCarCollision";
import IconCarGearbox from "./svg/IconCarGearbox";
import IconCarParking from "./svg/IconCarParking";
import IconCaravan from "./svg/IconCaravan";
import IconComponent from "./svg/IconComponent";
import IconDriving from "./svg/IconDriving";
import IconEletricMoped from "./svg/IconEletricMoped";
import IconFerryBoat from "./svg/IconFerryBoat";
import IconGasStation from "./svg/IconGasStation";
import IconGasStationOff from "./svg/IconGasStationOff";
import IconHambulance from "./svg/IconHambulance";
import IconMetro from "./svg/IconMetro";
import IconMoped from "./svg/IconMoped";
import IconMotor from "./svg/IconMotor";
import IconMotorbike1 from "./svg/IconMotorbike1";
import IconMotorbike2 from "./svg/IconMotorbike2";
import IconParking from "./svg/IconParking";
import IconRoad from "./svg/IconRoad";
import IconSailboat from "./svg/IconSailboat";
import IconScooter from "./svg/IconScooter";
import IconSmartCar from "./svg/IconSmartCar";
import IconSpeedometer from "./svg/IconSpeedometer";
import IconTruck from "./svg/IconTruck";
import IconTruckFast from "./svg/IconTruckFast";
import IconYacth from "./svg/IconYacth";

export const Icons = {
  airplane: IconAirplane,
  alarm: IconAlarm,
  anchor: IconAnchor,
  bus: IconBus,
  busStation: IconBusStation,
  car1: IconCar1,
  car2: IconCar2,
  carCollision: IconCarCollision,
  carGearbox: IconCarGearbox,
  carParking: IconCarParking,
  caravan: IconCaravan,
  component: IconComponent,
  driving: IconDriving,
  eletricMoped: IconEletricMoped,
  ferryBoat: IconFerryBoat,
  gasStation: IconGasStation,
  gasStationOff: IconGasStationOff,
  hambulance: IconHambulance,
  metro: IconMetro,
  moped: IconMoped,
  motor: IconMotor,
  motorbike1: IconMotorbike1,
  motorbike2: IconMotorbike2,
  parking: IconParking,
  road: IconRoad,
  sailboat: IconSailboat,
  scooter: IconScooter,
  smartCar: IconSmartCar,
  speedometer: IconSpeedometer,
  truck: IconTruck,
  truckFast: IconTruckFast,
  yacth: IconYacth,
} as const;

export type IconType = keyof typeof Icons;

export type IconProps = {
  name: IconType;
} & SvgIconProps;

export const Icon = ({ fontSize = "small", name, ...props }: IconProps) => {
  const IconElement = Icons[name];
  return <IconElement fontSize={fontSize} {...props} />;
};
