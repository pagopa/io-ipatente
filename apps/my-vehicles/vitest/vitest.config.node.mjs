import { configDefaults, defineConfig } from "vitest/config";
import tsconfigPaths from "vite-tsconfig-paths";

export default defineConfig({
  plugins: [tsconfigPaths()],
  test: {
    coverage: {
      exclude: [
        ...(configDefaults.coverage.exclude
          ? configDefaults.coverage.exclude
          : []),
        ".next/*",
        "mocks/*",
        "public/*",
        "*.config.js",
        "src/generated/**",
        "src/config/*",
        "src/types/*",
        "src/main.ts",
        "src/config.ts",
        "src/instrumentation.ts",
        "**/__mocks__/**",
        // FE
        "src/hooks/*",
        "src/layouts/*",
      ],
      extension: configDefaults.coverage.extension
        ?.toString()
        .replace(",.tsx", "")
        .split(","),
      ignoreEmptyLines: true,
      reporter: ["text", "json-summary", "json"],
      reportsDirectory: `${configDefaults.coverage.reportsDirectory}/backend`,
    },
    exclude: [
      ...configDefaults.exclude,
      "./vitest/vitest.setup.react.ts",
      "./vitest/vitest.setup.node.ts",
    ],
    include: ["./src/**/__tests__/*.ts"],
    setupFiles: ["./vitest/vitest.setup.node.ts"],
    typecheck: {
      ignoreSourceErrors: true,
    },
  },
});
