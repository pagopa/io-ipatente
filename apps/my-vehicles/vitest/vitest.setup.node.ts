/**
 * MSW configuration for Vitest (used as setupFile in vitest.config.node.ts)
 */
import { loadEnvConfig } from "@next/env";
import { afterAll, afterEach, beforeAll } from "vitest";

import { mswServerTest } from "../mocks/msw-server";

let mswServer: ReturnType<typeof mswServerTest>;
// Start server before all tests
beforeAll(() => {
  loadEnvConfig(process.cwd());
  // this done to correctly load the environment variables on msw handlers
  mswServer = mswServerTest();
  mswServer.listen({ onUnhandledRequest: "warn" });
});

//  Close server after all tests
afterAll(() => mswServer.close());

// Reset handlers after each test `important for test isolation`
afterEach(() => mswServer.resetHandlers());
