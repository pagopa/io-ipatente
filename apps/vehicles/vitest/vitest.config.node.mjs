import { loadEnv } from "vite";
import tsconfigPaths from "vite-tsconfig-paths";
import { configDefaults, defineConfig } from "vitest/config";

export default defineConfig({
  plugins: [tsconfigPaths()],
  test: {
    coverage: {
      exclude: [
        ...configDefaults.coverage.exclude,
        "**/__tests__",
        "src/generated/**",
        "src/app/api/auth/mocks/**",
        // FE
        "src/common/*",
        "src/hooks/*",
      ],
      ignoreEmptyLines: true,
      include: ["src/**/*.ts"],
      reporter: ["text", "html"],
      reportsDirectory: `${configDefaults.coverage.reportsDirectory}/backend`,
    },
    env: loadEnv("test", process.cwd(), ""),
    exclude: [
      ...configDefaults.exclude,
      "./vitest/vitest.setup.react.ts",
      "./vitest/vitest.setup.node.ts",
    ],
    include: ["./src/**/__tests__/*.ts"],
    setupFiles: ["./vitest/vitest.setup.node.ts"],
    typecheck: {
      ignoreSourceErrors: true,
    },
  },
});
