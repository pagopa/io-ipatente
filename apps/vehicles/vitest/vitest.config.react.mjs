/// <reference types="vitest" />
/// <reference types="vite/client" />

import react from "@vitejs/plugin-react";
import tsconfigPaths from "vite-tsconfig-paths";
import { configDefaults, defineConfig } from "vitest/config";
import { loadEnv } from "vite";

export default defineConfig({
  plugins: [react(), tsconfigPaths()],
  test: {
    coverage: {
      exclude: [
        ...(configDefaults.coverage.exclude
          ? configDefaults.coverage.exclude
          : []),
        ".next/*",
        "mocks/*",
        "public/*",
        "*.config.js",
        "src/generated/**",
        "src/config/*",
        "src/types/*",
        "src/main.ts",
        "src/config.ts",
        "src/instrumentation.ts",
        "**/__mocks__/**",
        // BE
        "src/app/*",
        "src/lib/be/*",
      ],
      ignoreEmptyLines: true,
      reporter: ["text", "json-summary", "json"],
      reportsDirectory: `${configDefaults.coverage.reportsDirectory}/frontend`,
    },
    css: true,
    env: loadEnv("test", process.cwd(), ""),
    environment: "jsdom",
    exclude: [
      ...configDefaults.exclude,
      "./vitest/vitest.setup.react.ts",
      "./vitest/vitest.setup.node.ts",
    ],
    globals: true,
    include: ["./src/**/*.test.tsx"],
    setupFiles: "./vitest/vitest.setup.react.ts",
    typecheck: {
      ignoreSourceErrors: true,
    },
  },
});
