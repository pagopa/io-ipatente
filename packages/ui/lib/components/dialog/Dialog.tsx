import {
  DialogContent,
  DialogTitle,
  IconButton,
  Dialog as MuiDialog,
  DialogProps as MuiDialogProps,
  Typography,
} from "@mui/material";
import { ReactNode } from "react";

import { Icon } from "../icon";

export type DialogProps = {
  body: ReactNode;
  onClose: () => void;
  title: string;
} & Pick<MuiDialogProps, "open">;

export const Dialog = ({ body, onClose, open, title }: DialogProps) => (
  <MuiDialog fullWidth={true} maxWidth="md" onClose={onClose} open={open}>
    <DialogTitle
      component="div"
      sx={{ alignItems: "center", display: "flex", p: 2 }}
    >
      <Typography variant="h6">{title}</Typography>
      <IconButton sx={{ ml: "auto" }}>
        <Icon color="primary" name="closeCircle" />
      </IconButton>
    </DialogTitle>
    <DialogContent dividers>{body}</DialogContent>
  </MuiDialog>
);
