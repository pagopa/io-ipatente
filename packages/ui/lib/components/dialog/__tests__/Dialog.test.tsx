import { Typography } from "@mui/material";
import { Stack } from "@mui/system";
import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";

import { Icon } from "../../icon";
import { Dialog, DialogProps } from "../Dialog";

const defaultProps: DialogProps = {
  body: (
    <Stack spacing={3}>
      <Stack alignItems="center">
        <Icon fontSize="large" name="success" />
      </Stack>
      <Typography textAlign="center" variant="body1">
        {
          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
        }
      </Typography>
    </Stack>
  ),
  onClose: () => null,
  open: true,
  title: "Dialog Test",
};

describe("Test Dialog Component", () => {
  it("Should match the snapshot with all props", () => {
    const comp = render(<Dialog {...defaultProps} />);
    expect(comp).toMatchSnapshot();
  });
});
