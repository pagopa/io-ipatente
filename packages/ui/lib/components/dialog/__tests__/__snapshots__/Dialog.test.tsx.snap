// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test Dialog Component > Should match the snapshot with all props 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body
    style="padding-right: 1024px; overflow: hidden;"
  >
    <div
      aria-hidden="true"
    />
    <div
      class="MuiDialog-root MuiModal-root css-zw3mfo-MuiModal-root-MuiDialog-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root MuiModal-backdrop css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-testid="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
        role="presentation"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        tabindex="-1"
      >
        <div
          aria-labelledby=":r0:"
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthMd MuiDialog-paperFullWidth css-rnmm7m-MuiPaper-root-MuiDialog-paper"
          role="dialog"
        >
          <div
            class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-p98fqf-MuiTypography-root-MuiDialogTitle-root"
            id=":r0:"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
            >
              Dialog Test
            </h6>
            <button
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-djcg5u-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeSmall css-1bptiuh-MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.0431 12L16.8516 11.1916C17.9653 10.0778 17.9653 8.26221 16.8516 7.14845C15.7378 6.03469 13.9222 6.03469 12.8084 7.14845L12 7.95689L11.1916 7.14845C10.0778 6.03469 8.26221 6.03469 7.14845 7.14845C6.03469 8.26221 6.03469 10.0778 7.14845 11.1916L7.95689 12L7.14845 12.8084C6.03469 13.9222 6.03469 15.7378 7.14845 16.8516L8.64 15.36L7.14845 16.8516C7.71488 17.418 8.45074 17.6894 9.17 17.6894C9.88926 17.6894 10.6251 17.418 11.1916 16.8516L10.1316 15.7916L11.1916 16.8516L12 16.0431L12.8084 16.8516C13.3749 17.418 14.1107 17.6894 14.83 17.6894C15.5493 17.6894 16.2851 17.418 16.8516 16.8516L15.36 15.36L16.8516 16.8516C17.9653 15.7378 17.9653 13.9222 16.8516 12.8084L16.0431 12ZM4.10938 12C4.10938 7.65498 7.65498 4.10938 12 4.10938C16.345 4.10938 19.8906 7.65498 19.8906 12C19.8906 16.345 16.345 19.8906 12 19.8906C7.65498 19.8906 4.10938 16.345 4.10938 12Z"
                  fill="currentColor"
                  stroke="currentColor"
                  stroke-width="4.21875"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <div
            class="MuiDialogContent-root MuiDialogContent-dividers css-1t4vnk2-MuiDialogContent-root"
          >
            <div
              class="MuiStack-root css-v2woy3-MuiStack-root"
            >
              <div
                class="MuiStack-root css-163ljxj-MuiStack-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-tzssek-MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 48 48"
                >
                  <path
                    d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z"
                    fill="#E6F9EF"
                  />
                  <path
                    d="M24.5715 14.5713C19.0615 14.5713 14.5715 19.0613 14.5715 24.5713C14.5715 30.0813 19.0615 34.5713 24.5715 34.5713C30.0815 34.5713 34.5715 30.0813 34.5715 24.5713C34.5715 19.0613 30.0815 14.5713 24.5715 14.5713ZM29.3515 22.2713L23.6815 27.9413C23.5415 28.0813 23.3515 28.1613 23.1515 28.1613C22.9515 28.1613 22.7615 28.0813 22.6215 27.9413L19.7915 25.1113C19.5015 24.8213 19.5015 24.3413 19.7915 24.0513C20.0815 23.7613 20.5615 23.7613 20.8515 24.0513L23.1515 26.3513L28.2915 21.2113C28.5815 20.9213 29.0615 20.9213 29.3515 21.2113C29.6415 21.5013 29.6415 21.9713 29.3515 22.2713Z"
                    fill="#1D7748"
                  />
                </svg>
              </div>
              <p
                class="MuiTypography-root MuiTypography-body1 css-1uwgr7b-MuiTypography-root"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        data-testid="sentinelEnd"
        tabindex="0"
      />
    </div>
  </body>,
  "container": <div
    aria-hidden="true"
  />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
